### PR TITLE
Auto-Download chromedriver

### PR DESCRIPTION
**Pull request by ShoGinn**
This is a quick modification to automatically download the most current chromedriver.. should eliminate lots of probs

### CODE REVIEW

File: ms_rewards.py [hunk 1]
```diff
@@ -119,13 +122,56 @@ def get_login_info():
         return json.load(f)
 
 
+def download_driver(driver_path, system):
+    # determine latest chromedriver version
+    url = "https://sites.google.com/a/chromium.org/chromedriver"
+    r = requests.get(url)
+    latest_version = max([float("{}.{}".format(version[0], version[1]))
```
Comment (xMNG) [inline ms_rewards.py]: Would it be better to pull the latest version by using their api vs a regex?
```
request = requests.get('https://chromedriver.storage.googleapis.com/LATEST_RELEASE')
request.text

# gives you '2.46', the latest version number.
```

File: ms_rewards.py [hunk 2]
```diff
@@ -134,11 +180,7 @@ def browser_setup(headless_mode, user_agent):
     options.add_argument('--no-sandbox')
     options.add_argument('--disable-dev-shm-usage')
 
-    "Windows, MacOS"
```
Comment (xMNG) [inline ms_rewards.py]: We wouldn't need these windows, macOS, or linux differentiators anymore right? Really good code.

File: ms_rewards.py [hunk 3]
```diff
@@ -119,13 +122,56 @@ def get_login_info():
...
+        url = "https://chromedriver.storage.googleapis.com/{}/chromedriver_win32.zip".format(latest_version)
+    elif system == "Darwin":
+        url = "https://chromedriver.storage.googleapis.com/{}/chromedriver_mac64.zip".format(latest_version)
+    elif system == "Linux":
+        url = "https://chromedriver.storage.googleapis.com/{}/chromedriver_linux64.zip".format(latest_version)
+
+    response = requests.get(url, stream=True)
+    zip_file_path = os.path.join(os.path.dirname(driver_path), os.path.basename(url))
+    handle = open(zip_file_path, "wb")
+    for chunk in response.iter_content(chunk_size=512):
```
Comment (xMNG) [inline ms_rewards.py]: Would it make sense to put a context manager here?
```
with open(zip_file_path, "wb") as handle:
    for chunk in response.iter_content(chunk_size=512):
           if chunk:  # filter out keep alive chunks
                 handle.write(chunk)
```

File: ms_rewards.py (hunk 1 above)
Comment (ShoGinn) [reply ms_rewards.py]: Lol.. the API would be better.. Much cleaner code than a regex

File: ms_rewards.py (hunk 2 above)
Comment (ShoGinn) [reply ms_rewards.py]: yeah the path is all that is necessary, and since the code knows where the chromedriver is...

File: ms_rewards.py (hunk 3 above)
Comment (ShoGinn) [reply ms_rewards.py]: fixed

File: ms_rewards.py (hunk 1 above)
Comment (ShoGinn) [reply ms_rewards.py]: updated to API

**Pull request closed without merging**